pkg/operations/kubernetesupgrade/upgrader.go

```diff
@@ -383,6 +385,43 @@ func (ku *Upgrader) upgradeAgentPools(ctx context.Context) error {
 }
 
 func (ku *Upgrader) upgradeAgentScaleSets(ctx context.Context) error {
+	if len(ku.ClusterTopology.AgentPoolScaleSetsToUpgrade) != 0 {
```

Comment:
This should probably be `> 0`, right?

Reply:
thanks for the comments, I will change it to > 0 in next iteration.